controllers/v1/platformUserRolesController.js

```diff
@@ -190,4 +190,31 @@ module.exports = class PlatformUserRoles extends Abstract {
     })
   }
 
+  /**
+  * @api {post} /user-management-service/api/v1/platformUserRoles/create add User
+  * @apiVersion 1.0.0
+  * @apiName create User Roles
```

Comment:
@rakeshSgr - create should be Create. Also is this API add User or create User Roles